### DISCUSSION TITLE
Strange error wire:click and class switch position cause it can't be clicked

### DISCUSSION THREAD

**Discussion by gzai** [livewire/livewire#3333] [7. Bugs (archived)] [answered]
Hi,

I have strange error. when I want to add data then delete and add data again, an error occurred can't click wire:click="createTvStation()". 
show inspect element, wire:click and class switch position. how to fix this error?

![strange-error-livewire](https://user-images.githubusercontent.com/4225733/125603616-9f9fa2cf-4df4-405b-80fc-57e5016cf05c.gif)

**Comment by ryatkins** [comment, marked as answer]
Any JavaScript console errors?

**Reply by gzai** [reply]
no errors

**Reply by ryatkins** [reply]
Can you try wrapping your TV stations section in an extra div and try putting keys on the TV divs so it differentiate the divs when the DOM diffing happens?

**Reply by gzai** [reply]
Yeah, adding extra div wrapping tv stations section solve the problem. thanks @ryatkins
